cmd/livekit-cli/room.go

```diff
@@ -43,19 +43,24 @@ var (
 					Required: true,
 				},
 				&cli.StringFlag{
-					Name:     "room-egress-file",
-					Usage:    "RoomCompositeRequest json file (see examples/room-composite-file.json)",
-					Required: false,
+					Name:  "room-egress-file",
+					Usage: "RoomCompositeRequest json file (see examples/room-composite-file.json)",
 				},
 				&cli.StringFlag{
-					Name:     "track-egress-file",
-					Usage:    "AutoTrackEgress json file (see examples/auto-track-egress.json)",
-					Required: false,
+					Name:  "participant-egress-file",
+					Usage: "ParticipantEgress json file (see examples/auto-participant-egress.json)",
+				},
+				&cli.StringFlag{
+					Name:  "track-egress-file",
+					Usage: "AutoTrackEgress json file (see examples/auto-track-egress.json)",
 				},
 				&cli.UintFlag{
-					Name:     "min-playout-delay",
-					Usage:    "minimum playout delay",
-					Required: false,
+					Name:  "min-playout-delay",
```

Comment:
should we add a `max-playout-delay` support as well given that it is supported now?

Reply:
perhaps as a separate PR.. wanted to keep this one tight.